corehq/apps/domain/forms.py

```diff
@@ -1165,39 +1168,40 @@ def __init__(self, account, domain, creating_user, plan_version, current_subscri
         )
 
     def save(self, commit=True):
```

Comment:
:+1:

On Mon, Nov 30, 2015 at 10:40 AM, Nick Pellegrino <notifications@github.com>
wrote:

> In corehq/apps/domain/forms.py
> <https://github.com/dimagi/commcare-hq/pull/9320#discussion_r46167518>:
>
> > @@ -1165,39 +1168,40 @@ def __init__(self, account, domain, creating_user, plan_version, current_subscri
> >          )
> >
> >      def save(self, commit=True):
>
> not quite - the exception gets swallowed, so if you decorate the function
> with transaction.atomic, it will always end up committing the outstanding
> writes. By putting the with inside the try, if an exception is raised,
> the transaction will be aborted and after that the exception will be caught.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9320/files#r46167518>.
>